aou/aou-rstudio-bioconductor/rstudio/Dockerfile

```diff
@@ -0,0 +1,137 @@
+
+## Install specific version of BiocManager
+ADD scripts/install.R /tmp/
+RUN R -f /tmp/install.R
+
+# DEVEL: Add sys env variables to DEVEL image
+# Variables in Renviron.site are made available inside of R.
+# Add libsbml CFLAGS
+RUN curl -O http://bioconductor.org/checkResults/devel/bioc-LATEST/Renviron.bioc \
+    && sed -i '/^IS_BIOC_BUILD_MACHINE/d' Renviron.bioc \
+    && cat Renviron.bioc | grep -o '^[^#]*' | sed 's/export //g' >>/etc/environment \
+    && cat Renviron.bioc >> /usr/local/lib/R/etc/Renviron.site \
+    && echo BIOCONDUCTOR_VERSION=${BIOCONDUCTOR_VERSION} >> /usr/local/lib/R/etc/Renviron.site \
+    && echo BIOCONDUCTOR_DOCKER_VERSION=${BIOCONDUCTOR_DOCKER_VERSION} >> /usr/local/lib/R/etc/Renviron.site \
+    && echo 'LIBSBML_CFLAGS="-I/usr/include"' >> /usr/local/lib/R/etc/Renviron.site \
+    && echo 'LIBSBML_LIBS="-lsbml"' >> /usr/local/lib/R/etc/Renviron.site \
+    && rm -rf Renviron.bioc
+
+ENV LIBSBML_CFLAGS="-I/usr/include"
+ENV LIBSBML_LIBS="-lsbml"
+ENV BIOCONDUCTOR_DOCKER_VERSION=$BIOCONDUCTOR_DOCKER_VERSION
+ENV BIOCONDUCTOR_VERSION=$BIOCONDUCTOR_VERSION
+
+## Use binary repos after
+ENV BIOCONDUCTOR_USE_CONTAINER_REPOSITORY=TRUE
+
+# ANVIL
+RUN apt-get update && apt-get install -yq --no-install-recommends \
+    gnupg \
+    lsb-release \
+ && echo "deb http://packages.cloud.google.com/apt cloud-sdk main" > /etc/apt/sources.list.d/google-cloud-sdk.list \
+ && curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add - \
+ && apt-get update \
+ && apt-get install -yq --no-install-recommends \
+    google-cloud-sdk \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
+
+RUN pip3 -V \
+    && pip3 install --no-cache-dir -U crcmod
+
+# add rstudio user to users group
+RUN usermod -g users rstudio \
+    && useradd -m -s /bin/bash -N -u 1001 welder-user
+
+# google-cloud R packages
+## Install specific version of BiocManager
+RUN R -f /rocker_scripts/install_bioconductor.R
+
+ENV RSTUDIO_HOME /etc/rstudio
+ENV RSTUDIO_USERSETTING /home/rstudio/.config/rstudio/rstudio-prefs.json
+
+ADD rserver.conf $RSTUDIO_HOME/rserver.conf
+
+RUN sed -i 's/"always_save_history": false/"always_save_history": true/g' $RSTUDIO_USERSETTING \
+ && sed -i 's/"save_workspace": "never"/"save_workspace": "always"/g' $RSTUDIO_USERSETTING
+
+COPY user_scripts ${RSTUDIO_HOME}/scripts
+RUN find ${RSTUDIO_HOME}/scripts -name '*.sh' -type f | xargs chmod +x \
+ && echo "PIP_USER=true" >>  /usr/local/lib/R/etc/Renviron.site
+
+ENV DISABLE_AUTH=true
+ENV ROOT=false
+
+# Restrict download speed to 8Mbit/s = 1MB/s and upload speeds to 4Mbit/s = 512KB/s
+# Wondershaper will be executed as a different user compared to the rstudio user
+# in /init.  Should not be possible to reset Wondershaper to unlimited speeds or delete
+# the Wondershaper executable.
+# TODO: Add VPC ip addresses to whitelist
+CMD /bin/bash -c "wondershaper eth0 8096 4048 && nginx -g 'daemon on;' && /init"
```

Comment:
Are we using the default Nginx config file?

Reply:
We don't actually need nginx to be running.  We just needed the kernel modules and settings and configs from that docker image